src/test/resources/data/discover/bsearch.json

```diff
@@ -0,0 +1,111 @@
+{
+  "batch":[
+    {
+      "request":{
+        "params":{
+          "index":"kibana_sample_data_ecommerce",
+          "body":{
+            "size":500,
+            "sort":[
+              {
+                "order_date":{
+                  "order":"desc",
+                  "unmapped_type":"boolean"
+                }
+              }
+            ],
+            "version":true,
+            "fields":[
+              {
+                "field":"*",
+                "include_unmapped":"true"
+              },
+              {
+                "field":"customer_birth_date",
+                "format":"strict_date_optional_time"
+              },
+              {
+                "field":"order_date",
+                "format":"strict_date_optional_time"
+              },
+              {
+                "field":"products.created_on",
+                "format":"strict_date_optional_time"
+              }
+            ],
+            "aggs":{
+              "2":{
+                "date_histogram":{
+                  "field":"order_date",
+                  "${intervalName}":"${intervalValue}",
+                  "time_zone":"Europe/Berlin",
+                  "min_doc_count":1
+                }
+              }
+            },
+            "script_fields":{
+
+            },
+            "stored_fields":[
+              "*"
+            ],
+            "runtime_mappings":{
+
+            },
+            "_source":false,
+            "query":{
+              "bool":{
+                "must":[
+
+                ],
+                "filter":[
+                  {
+                    "match_all":{
+
+                    }
+                  },
+                  {
+                    "range":{
+                      "order_date":{
+                        "gte":"${startTime}",
+                        "lte":"${endTime}",
+                        "format":"strict_date_optional_time"
+                      }
+                    }
+                  }
+                ],
+                "should":[
+
+                ],
+                "must_not":[
+
+                ]
+              }
+            },
+            "highlight":{
+              "pre_tags":[
+                "@kibana-highlighted-field@"
+              ],
+              "post_tags":[
+                "@/kibana-highlighted-field@"
+              ],
+              "fields":{
+                "*":{
+
+                }
+              },
+              "fragment_size":2147483647
+            }
+          },
+          "preference":"${preference}"
```

Comment:
I'm not sure how critical to have it, but now every virtual user will have a timestamp value here